bench-vortex/build.rs

```diff
@@ -22,9 +22,9 @@ use std::path::PathBuf;
 /// static lib is not self-contained. This means that it includes symbols which
 /// are not defined as part of the static library.
 fn main() {
-    const DUCKDB_VERSION: &str = "v1.3.1";
+    const DUCKDB_VERSION: &str = "v1.3.2";
```

Comment:
Can we just put the library inside a `duckdb-lib` directory and add that to the lib_path? These constants will make automated upgrades annoying